<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgEditFurniture</class>
 <widget class="QDialog" name="dlgEditFurniture">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>235</width>
    <height>303</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>235</width>
    <height>303</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>235</width>
    <height>303</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <property name="margin">
    <number>6</number>
   </property>
   <property name="spacing">
    <number>3</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="lbName">
     <property name="text">
      <string>Nazwa:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLineEdit" name="edName"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lbColor">
     <property name="text">
      <string>Kolor:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <layout class="QHBoxLayout" name="layHorizontal">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QFrame" name="frmColor">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnColor">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QGroupBox" name="gbSize">
     <property name="title">
      <string>Rozmiar</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="lbSizeWidth">
        <property name="text">
         <string>Szerokość:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="sbSizeWidth"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lbSizeHeight">
        <property name="text">
         <string>Wysokość:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="sbSizeHeight"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QGroupBox" name="gbMargins">
     <property name="title">
      <string>Margines</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="lbMarginsHorizontal">
        <property name="text">
         <string>Poziomy:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="sbMarginsHorizontal"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lbMarginsVertical">
        <property name="text">
         <string>Pionowy:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="sbMarginsVertical"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <layout class="QHBoxLayout" name="layHorizontal2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnOK">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Anuluj</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QGroupBox" name="gbContent">
     <property name="title">
      <string>Przechowuje</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="lbContentType">
        <property name="text">
         <string>Rodzaj:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="cbContentType">
        <item>
         <property name="text">
          <string>Rzeczy</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Ludzie</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lbContentLimit">
        <property name="text">
         <string>Ilość:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="sbContentLimit"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>edName</tabstop>
  <tabstop>btnColor</tabstop>
  <tabstop>sbSizeWidth</tabstop>
  <tabstop>sbSizeHeight</tabstop>
  <tabstop>cbContentType</tabstop>
  <tabstop>sbContentLimit</tabstop>
  <tabstop>sbMarginsHorizontal</tabstop>
  <tabstop>sbMarginsVertical</tabstop>
  <tabstop>btnOK</tabstop>
  <tabstop>btnCancel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>dlgEditFurniture</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>183</x>
     <y>279</y>
    </hint>
    <hint type="destinationlabel">
     <x>35</x>
     <y>279</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
